sterling/transfer.py

```diff
@@ -6,7 +6,8 @@
 
 class Transfer(Sterling):
     def InterbankNameEnquiry(self, query):
-        url = urljoin(BASE_URL(self.url), "/sterling/TransferAPIs/api/Spay/InterbankNameEnquiry")
+        url = urljoin(BASE_URL(self.url),
+                      "/sterling/TransferAPIs/api/Spay/InterbankNameEnquiry")
```

Comment:
What's the problem with having it oneline

Reply:
I'm using a linter